Overview page for Run your dbt projects

nghi-ly opened the pull request:
## Description & motivation
New docs page that provides overview information on running dbt projects by rewriting this page:

https://docs.getdbt.com/docs/building-a-dbt-project/projects

Redirect was handled in PR #2121 

Closes #2076 

## Checklist
If you added new pages (delete if not applicable):
- [x] The page has been added to `website/sidebars.js`
- [ ] The new page has a unique filename

nghi-ly committed:
Overview page for Run your dbt projects

nghi-ly committed:
Folded in review feedback

nghi-ly committed:
Add link to create project docs

nghi-ly commented on website/docs/docs/develop/run-your-dbt-projects.md:
```diff
@@ -0,0 +1,22 @@
+---
+You can run your dbt projects with [dbt Cloud](/docs/develop/dbt-cloud-features) and [dbt Core](https://github.com/dbt-labs/dbt-core). dbt Cloud is a hosted application where you can develop directly from a web browser. dbt Core is an open source project where you can develop from the command line.
+
+Among other features, dbt Cloud provides a hosted development environment to help you build, test, run, and version control your project faster. It also includes a purpose-built scheduler and an easier way to share your [dbt project's documentation](/docs/collaborate/build-and-view-your-docs) with your team. These development tasks are directly built into dbt Cloud. See [dbt Cloud IDE] (link to o ...
```
hey @mirnawong1 : i think more links to IDE docs could be added in this paragraph. let's chat more! 🙏

mirnawong1 commented on website/docs/docs/develop/run-your-dbt-projects.md:
```diff
@@ -0,0 +1,22 @@
+---
+You can run your dbt projects with [dbt Cloud](/docs/develop/dbt-cloud-features) and [dbt Core](https://github.com/dbt-labs/dbt-core). dbt Cloud is a hosted application where you can develop directly from a web browser. dbt Core is an open source project where you can develop from the command line.
+
+Among other features, dbt Cloud provides a hosted development environment to help you build, test, run, and version control your project faster. It also includes a purpose-built scheduler and an easier way to share your [dbt project's documentation](/docs/collaborate/build-and-view-your-docs) with your team. These development tasks are directly built into dbt Cloud. See [dbt Cloud IDE] (link to o ...
```
updated links and sugg IDE instead of hosted

```suggestion
Among other features, dbt Cloud provides an integrated development environment (IDE) to help you build, test, run, and version control your project faster. It also includes a purpose-built scheduler and an easier way to share your [dbt project's documentation](/docs/collaborate/build-and-view-your-docs) with your team. These development tasks are directly built into dbt Cloud. Review [dbt Cloud IDE] (/docs/develop/develop-in-the-cloud) for more details.
```

mirnawong1 replied:
(/docs/develop/develop-in-the-cloud) changing to this link now

mirnawong1 commented on website/docs/docs/develop/run-your-dbt-projects.md:
```diff
@@ -0,0 +1,22 @@
+
+With dbt Core, you can run your dbt projects from the command line. The command line interface (CLI) is available from your computer's terminal application such as Terminal and iTerm. When using the command line, you can run commands and do other work from the current working directory on your computer. Before running the dbt project from the command line, make sure you are working in your dbt pr ...
+
+When working with a project, the dbt commands you commonly use are:
```
i'm suggesting being explicit here with the cli and ide

```suggestion
When working with a project in the CLI or Cloud IDE, the dbt commands you commonly use are:
```

nghi-ly replied:
i don't think the sentence is confusing or ambiguous without "in the CLI or Cloud IDE" so wanna lean toward being concise (fewer words). 
 
do you find it unclear?

mirnawong1 replied:
oh yea i did initially - i think it's because it followed the dbt core summary maybe? so i had to re read to make sure the commands were related to both

mirnawong1 commented on website/docs/docs/develop/run-your-dbt-projects.md:
```diff
@@ -0,0 +1,22 @@
+## Related docs
+
+- [How we set up our computers for working on dbt projects](https://discourse.getdbt.com/t/how-we-set-up-our-computers-for-working-on-dbt-projects/243)
+- [Model selection syntax](https://docs.getdbt.com/reference/node-selection/syntax)
```
suggesting some more docs on the scheduler (i linked the dbt cloud features page - is this right?), ide, version control. 

```suggestion
- [Model selection syntax](https://docs.getdbt.com/reference/node-selection/syntax)
- [Version control basics](/docs/collaborate/git/version-control-basics)
- [dbt Cloud tips](/docs/develop/dbt-cloud-tips)
```

nghi-ly committed:
Update website/docs/docs/develop/run-your-dbt-projects.md

Co-authored-by: mirnawong1 <89008547+mirnawong1@users.noreply.github.com>

nghi-ly committed:
Folded in review from meeting

nghi-ly commented on website/docs/docs/develop/run-your-dbt-projects.md:
```diff
@@ -0,0 +1,23 @@
+
+- [How we set up our computers for working on dbt projects](https://discourse.getdbt.com/t/how-we-set-up-our-computers-for-working-on-dbt-projects/243)
+- [Model selection syntax](https://docs.getdbt.com/reference/node-selection/syntax)
+- [Develop in the Cloud](/docs/develop/develop-in-the-cloud)
```
@mirnawong1 , added the "Develop in the Cloud" to related docs

nghi-ly commented on website/docs/docs/develop/run-your-dbt-projects.md:
```diff
@@ -0,0 +1,23 @@
+---
+You can run your dbt projects with [dbt Cloud](/docs/develop/dbt-cloud-features) and [dbt Core](https://github.com/dbt-labs/dbt-core). dbt Cloud is a hosted application where you can develop directly from a web browser. dbt Core is an open source project where you can develop from the command line.
+
+Among other features, dbt Cloud provides a development environment to help you build, test, run, and [version control](/docs/collaborate/git-version-control) your project faster. It also includes an easier way to share your [dbt project's documentation](/docs/collaborate/build-and-view-your-docs) with your team. These development tasks are directly built into dbt Cloud for an _integrated developm ...
```
hi @mirnawong1 , this paragraph is introducing the IDE so would like to link out to more information about all the features in the IDE. i think the "Develop in the Cloud" is helpful to reference in this page too; i've added it to the "Related docs" section as an escape hatch to it

is `/docs/develop/dbt-cloud-features#ide-features` the correct anchor link to use for the **IDE features** section?

mirnawong1 replied:
/docs/develop/dbt-cloud-features.md#ide-features is correct

nghi-ly commented:
hey @mirnawong1 , tysm for turning this around so quickly!! my changes are:

- worked on the wording in a couple paragraphs so hopefully it reads smoother
- added more link outs in the second paragraph to provide more escape hatches for deeper dives 
- added a new link to "Related docs" section

please review my latest draft when you get a chance! thanks!

nghi-ly committed:
Merge branch 'next' into ly-docs-run-your-project

mirnawong1 reviewed (approved):
hi @nghi-ly , looks good on my end!

nghi-ly committed:
Switch IDE links back

nghi-ly committed:
Merge branch 'next' into ly-docs-run-your-project

nghi-ly committed:
Merge branch 'next' into ly-docs-run-your-project

runleonarun commented on website/docs/docs/develop/run-your-dbt-projects.md:
```diff
@@ -0,0 +1,23 @@
+
+With dbt Core, you can run your dbt projects from the command line. The command line interface (CLI) is available from your computer's terminal application such as Terminal and iTerm. When using the command line, you can run commands and do other work from the current working directory on your computer. Before running the dbt project from the command line, make sure you are working in your dbt pr ...
+
+When you run your dbt project, the commands you commonly use are:
```
This felt a little Core focused, but it's true for either product. Maybe a tiny tweak will clarify
```suggestion
When running your dbt project from dbt Core or dbt Cloud, you'll commonly use:
```

nghi-ly replied:
```suggestion
When running your project from dbt Core or dbt Cloud, the commands you commonly use are:
```

nghi-ly replied:
@mirnawong1 also wanted to extend the first clause! 🙇‍♀️

runleonarun reviewed (changes requested):
@nghi-ly Just one tiny suggestion!

nghi-ly committed:
Update website/docs/docs/develop/run-your-dbt-projects.md

settling on this after chatting with runleonarun on slack

runleonarun reviewed (approved):
🚢 it!
